src/hotel-management-app/src/app/customer/cart/cart.component.html

```diff
@@ -12,22 +12,24 @@
         <mat-card-content>
```

Comment:
In Cart component there is additional left-right scroll could you remove it

Reply:
resolved